.github/workflows/sync.yml

```diff
@@ -0,0 +1,35 @@
+name: Sync metadata from contrib
+
+on:
+ workflow_dispatch:
+ 
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  sync_metadata:
+    name: Sync metadata
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+      - uses: actions/checkout@v3
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.20'
+      - name: Checkout new branch
+        run: git checkout -b metadata-update-${{ steps.date.outputs.date }}
+      - name: Download settings from contrib
+        run: |
+          cd cfgschema
+          go run .
+      - name: Download metrics from contrib
```

Comment:
This is fine for now, but we may eventually want to split these two operations into two different workflows.